doc/compile_dpdk_with_musl_in_composite.md

```diff
@@ -0,0 +1,34 @@
+## How to compile DPDK with Musl libc in Composite?
+
+### Remove shared library support in DPDK meson build system
```

Comment:
I don't understand this documentation. Isn't this already done in the submodule? Why are there instructions on doing it here? Are these to document what you've already done? If that's the case, it doesn't make much sense to write this in the style of instructions.

Reply:
My original thought about this is that we need documentations for other maintainers. They might not be able to understand what we hacked within DPDK and do not understand why we did that. This is also for future DPDK updates in case that some parts within DPDK no longer work anymore, we can find some clues in this documentation.